openedx/features/course_experience/tests/views/test_welcome_message.py

```diff
@@ -50,66 +39,36 @@ def dismiss_message_url(course):
     return reverse(
         'openedx.course_experience.dismiss_welcome_message',
         kwargs={
-            'course_id': six.text_type(course.id),
+            'course_id': str(course.id),
         }
     )
 
 
 @ddt.ddt
-class TestWelcomeMessageView(ModuleStoreTestCase):
+class TestWelcomeMessageView(CourseUpdatesTestCase):
     """
     Tests for the course welcome message fragment view.
 
     Also tests the LatestUpdate view because the functionality is similar.
     """
-    def setUp(self):
-        """Set up the simplest course possible, then set up and enroll our fake user in the course."""
-        super(TestWelcomeMessageView, self).setUp()
-        with self.store.default_store(ModuleStoreEnum.Type.split):
-            self.course = CourseFactory.create()
-            with self.store.bulk_operations(self.course.id):
-                # Create a basic course structure
-                chapter = ItemFactory.create(category='chapter', parent_location=self.course.location)
-                section = ItemFactory.create(category='sequential', parent_location=chapter.location)
-                ItemFactory.create(category='vertical', parent_location=section.location)
-        self.user = UserFactory(password=TEST_PASSWORD)
-        CourseEnrollment.enroll(self.user, self.course.id)
-        self.client.login(username=self.user.username, password=TEST_PASSWORD)
-
-    def tearDown(self):
-        remove_course_updates(self.user, self.course)
-        super(TestWelcomeMessageView, self).tearDown()
-
     @ddt.data(welcome_message_url, latest_update_url)
     def test_message_display(self, url_generator):
-        create_course_update(self.course, self.user, 'First Update', date='January 1, 2000')
-        create_course_update(self.course, self.user, 'Second Update', date='January 1, 2017')
-        create_course_update(self.course, self.user, 'Retroactive Update', date='January 1, 2010')
+        self.create_course_update('First Update', date='January 1, 2000')
+        self.create_course_update('Second Update', date='January 1, 2017')
+        self.create_course_update('Retroactive Update', date='January 1, 2010')
         response = self.client.get(url_generator(self.course))
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, 'Second Update')
         self.assertContains(response, 'Dismiss')
 
-    @ddt.data(welcome_message_url, latest_update_url)
-    def test_replace_urls(self, url_generator):
```

Comment:
Moved to my new test file for just the utility funtions themselves. This didn't feel like a view test.